pkg/controller/servicemesh/memberroll/controller.go

```diff
@@ -741,7 +741,7 @@ func (r *namespaceReconciler) reconcileNamespaceInMesh(namespace string) error {
 			err = r.client.Update(context.TODO(), namespaceResource)
 			if err != nil {
 				if errors.IsConflict(err) {
-					namespaceResource := &corev1.Namespace{}
+					namespaceResource = &corev1.Namespace{}
```

Comment:
Can probably remove this line altogether.

Reply:
Probably, and that was my gut instinct, but I wasn't sure whether or not the client would clear all the fields.  (I know for SAR in member roll validator, it doesn't clear all the status fields.)

Reply:
Ah, ok. Then better not remove it for now.